package.json

```
@@ -1,13 +1,20 @@
 {
   "name": "pejuangkode.com",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Sebelum kita mulai rilis di https://pejuangkode.com, sementara versinya biar tetep `0.0.1` dulu saja Mas. 😁 
```suggestion
  "version": "0.0.1",
```

---

Yes correct 😁